src/chigame/games/models.py

```diff
@@ -61,8 +61,24 @@ class Player(models.Model):
     A player in a match.
     """
 
+    WIN = 0
+    DRAW = 1
+    LOSE = 2
+    WITHDRAWAL = 3
+
+    OUTCOMES = (
+        (WIN, "win"),
+        (DRAW, "draw"),
+        (LOSE, "lose"),
+        (WITHDRAWAL, "withdrawal"),
+    )
+
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     match = models.ForeignKey(Match, on_delete=models.CASCADE)
+    team = models.TextField(blank=True, null=True)
+    role = models.TextField(blank=True, null=True)
+    outcome = models.PositiveSmallIntegerField(choices=OUTCOMES)
```

Comment:
This should be nullable in case a player is currently in a match that hasn't been completed yet.